README.md

```diff
@@ -121,16 +137,16 @@ not required):
        "irods_default_resource" : "demoResc"
    }
    ```
-   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request.
+   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request. Be sure to set the *irods_default_resource*, this variable is not set when you create the file with *iinit* or when you copy it over from another user.
    
 2. As the user who runs the GridFTP server, try an `ils` icommand to verify that 
    the information set in the *irods_environment.json* are fine. If needed, perform 
    an `iinit` to authenticate the iRODS user. 
 
-2. Add the following lines to the GridFTP configuration file ( tipically 
-   *$GLOBUS_LOCATION/etc/gridftp.conf* ):
+3. Add the following lines to the GridFTP configuration file (typically 
+   *$GLOBUS_LOCATION/etc/gridftp.conf*):
    ```
-   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI"
+   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI/B2STAGE-GridFTP/"
```

Comment:
That is correct.

Here some output of the installation (the B2STAGE-GridFTP folder is conveniently placed in the iRODS_DSI folder):

```sh
~/iRODS_DSI/B2STAGE-GridFTP$ ls ../
B2STAGE-GridFTP 
```

```sh
~/iRODS_DSI/B2STAGE-GridFTP$ ls
ChangeLog.txt        CMakeLists.txt              images                setup.sh
CMakeCache.txt       DSI                         install_manifest.txt  setup.sh.template
CMakeFiles           external                    Makefile              cmake_install.cmake  gridmap_iRODS_callout.conf  README.md
```
```sh
~/iRODS_DSI/B2STAGE-GridFTP$ source setup.sh
~/iRODS_DSI/B2STAGE-GridFTP$ cmake CMakeLists.txt
GLOBUS_LOCATION is defined as /usr
IRODS_PATH is defined as /usr
CMake Warning at CMakeLists.txt:87 (message):
  RESOURCE_MAP_PATH is NOT defined!


CMake Warning at CMakeLists.txt:96 (message):
  FLAVOR is NOT defined!


DEST_LIB_DIR is defined as /home/mallet/iRODS_DSI
DEST_ETC_DIR is defined as /home/mallet/iRODS_DSI
DEST_BIN_DIR is defined as /home/mallet/iRODS_DSI
DSI library name will be: globus_gridftp_server_iRODS
Gridmap callout library name will be: gridmap_iRODS_callout
-- Configuring done
-- Generating done
-- Build files have been written to: /home/mallet/iRODS_DSI/B2STAGE-GridFTP
```

```sh
~/iRODS_DSI/B2STAGE-GridFTP$ make install
Linking CXX shared library libglobus_gridftp_server_iRODS.so
[ 30%] Built target globus_gridftp_server_iRODS
Linking CXX shared library libgridmap_iRODS_callout.so
[ 50%] Built target gridmap_iRODS_callout
[ 70%] Built target testirodsmap
[100%] Built target testpidmanager
Install the project...
-- Install configuration: ""
-- Installing: /home/mallet/iRODS_DSI/libglobus_gridftp_server_iRODS.so.1.8
-- Installing: /home/mallet/iRODS_DSI/libglobus_gridftp_server_iRODS.so
-- Installing: /home/mallet/iRODS_DSI/libgridmap_iRODS_callout.so.1.8
-- Installing: /home/mallet/iRODS_DSI/libgridmap_iRODS_callout.so
-- Up-to-date: /home/mallet/iRODS_DSI/testirodsmap
-- Up-to-date: /home/mallet/iRODS_DSI/testpidmanager
-- Up-to-date: /home/mallet/iRODS_DSI/gridmap_iRODS_callout.conf
```
```sh
~/iRODS_DSI/B2STAGE-GridFTP$ ls
ChangeLog.txt        gridmap_iRODS_callout.conf             Makefile
CMakeCache.txt       images                                 README.md
CMakeFiles           install_manifest.txt                   setup.sh
cmake_install.cmake  libglobus_gridftp_server_iRODS.so      setup.sh.template
CMakeLists.txt       libglobus_gridftp_server_iRODS.so.1.8  testirodsmap
DSI                  libgridmap_iRODS_callout.so            testpidmanager
external             libgridmap_iRODS_callout.so.1.8
```
```sh
~/iRODS_DSI/B2STAGE-GridFTP$ ls ../
B2STAGE-GridFTP                    libglobus_gridftp_server_iRODS.so.1.8  testirodsmap
gridmap_iRODS_callout.conf         libgridmap_iRODS_callout.so            testpidmanager
libglobus_gridftp_server_iRODS.so  libgridmap_iRODS_callout.so.1.8
```